.vib/seaweedfs/vib-verify.json

```diff
@@ -50,6 +50,21 @@
               "OS"
             ]
           }
+        },
+        {
+          "action_id": "goss",
```

Comment:
The default entrypoint/cmd of the container keeps it up, right? No need for custom runtime parameters?

Reply:
Good catch! I forgot it